server/engine/src/main/java/org/eclipse/lsp/cobol/core/engine/symbols/SymbolAccumulator.java

```diff
@@ -59,28 +56,10 @@ public class SymbolAccumulator implements VariableAccumulator {
    * @param programNode the program where this variable belongs to.
    * @param node the variable definition node
    */
-  public void addVariableDefinition(ProgramNode programNode, VariableNode node) {
+  public void addVariable(ProgramNode programNode, VariableNode node) {
     createOrGetSymbolTable(programNode).register(node);
   }
 
-  /**
-   * * Register variable definitions into nearest ProgramNode
-   *
-   * @param node the node with VariableDefinitionNodes
-   */
-  public void registerVariablesInProgram(Node node) {
```

Comment:
Why did you removed/inlined this function everywhere?

Reply:
It didn't make sense to me to increase api surface so we can traverse some subtree to extract variables definitions. In my head it should be done earlier in processing.